pkg/meta/utils.go

```diff
@@ -553,3 +558,34 @@ func (m *baseMeta) getTreeSummary(ctx Context, tree *TreeSummary, depth, topN ui
 	}
 	return 0
 }
+
+func (m *baseMeta) atimeNeedsUpdate(attr *Attr, now time.Time) bool {
+	if m.conf.AtimeMode == RelAtime && relatimeNeedUpdate(attr, now) {
+		return true
+	}
+
+	return m.conf.AtimeMode == StrictAtime && !now.Equal(time.Unix(attr.Atime, int64(attr.Atimensec)))
```

Comment:
Maybe we don't need that strict even in `StrictAtime` mode to reduce updates, i.e, compare `attr.Atime` without `attr.Atimensec` should be enough.

Reply:
Ok, I changed it to update atime only when last update is more than 1s ago.